sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/AboutDirectorySteps.java

```diff
@@ -0,0 +1,160 @@
+package org.sormas.e2etests.steps.web.application;
+
+import static org.sormas.e2etests.pages.application.AboutPage.DATA_DICTIONARY_BUTTON;
+import static org.sormas.e2etests.pages.application.users.CreateNewUserPage.LANGUAGE_COMBOBOX;
+import static org.sormas.e2etests.pages.application.users.CreateNewUserPage.SAVE_BUTTON;
+
+import com.detectlanguage.DetectLanguage;
+import com.google.inject.Inject;
+import cucumber.api.java8.En;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.CellType;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.ss.usermodel.Sheet;
+import org.apache.poi.ss.usermodel.Workbook;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+import org.sormas.e2etests.helpers.WebDriverHelpers;
+import org.testng.asserts.SoftAssert;
+
+public class AboutDirectorySteps implements En {
+  public static final String userDirPath = System.getProperty("user.dir");
+  public static final List<String> xlsxFileContentList = new ArrayList<>();
+  public static String language;
+
+  @Inject
+  public AboutDirectorySteps(WebDriverHelpers webDriverHelpers, SoftAssert softly) {
+
+    When(
+        "I select {string} language from Combobox in User settings",
+        (String chosenLanguage) -> {
+          language = chosenLanguage;
+          webDriverHelpers.selectFromCombobox(LANGUAGE_COMBOBOX, chosenLanguage);
+          webDriverHelpers.clickOnWebElementBySelector(SAVE_BUTTON);
+        });
+
+    When(
```

Comment:
this method is not working. You have a switch, but there's no parameter passed there.